pkg/plugins/golang/v4/scaffolds/init.go

```diff
@@ -162,5 +162,8 @@ func (s *initScaffolder) Scaffold() error {
 		&e2e.Test{},
 		&e2e.SuiteTest{},
 		&utils.Utils{},
+		&templates.DevContainerDockerfile{},
+		&templates.DevContainer{},
+		&templates.DevContainerPostInstallScript{},
```

Comment:
I think we do not need this one. I will comment bellow.

Reply:
@camilamacedo86 makes total sense! This will also simplify the my other wise GH Action that I've implemented so far (it's throwing some error related with  NAT table setup while running `kind create cluster` command).

Reply:
We probably could check it via GitHub action to ensure that all will be fine within the devcontainer and users will be able to run kind create